src/Utils.js

```diff
@@ -213,6 +213,19 @@ export function coercePropValue(propName, propValue) {
     return propValue;
   }
 
+  // can be the empty string
```

Comment:
It didn't feel right having `' '` be treated as the empty string, or `'   NaN'` be treated as a literal NaN - I think the current behavior of throwing for those cases is appropriate.